fcrepo-kernel/src/main/java/org/fcrepo/kernel/observer/FedoraEvent.java

```diff
@@ -184,6 +192,13 @@ public long getDate() {
         }
     }
 
+    /**
+     * Get the event ID.
+    **/
+    public String getEventID() {
```

Comment:
getIdentifier() returns the underlying JCR event's identifier -- i.e., the path of the node that's being acted on.  I'll update the Javadoc to make this more clear.